crml/governance/Cargo.toml

```diff
@@ -11,9 +11,20 @@ description = "A runtime module for decentralized governance of the CENNZnet pro
 codec = { version = "2.0.0", package = "parity-scale-codec", default-features = false }
 serde = { version = "1.0.102", optional = true }
 cennznet-primitives = { path = "../../primitives", default-features = false }
-frame-support = { git = "https://github.com/cennznet/substrate", rev = "d05b61e9b948621416a3c6eea2e10fd2f1b62aba", default-features = false }
-frame-system = { git = "https://github.com/cennznet/substrate", rev = "d05b61e9b948621416a3c6eea2e10fd2f1b62aba", default-features = false }
-sp-std = { git = "https://github.com/cennznet/substrate", rev = "d05b61e9b948621416a3c6eea2e10fd2f1b62aba", default-features = false }
+frame-support = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false }
+frame-system = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false }
+sp-std = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false }
+pallet-identity = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false }
+log = { version = "0.4.14", default-features = false }
+sp-runtime = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false  }
+sp-core = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false  }
+sp-staking = { git = "https://github.com/cennznet/substrate", default-features = false, rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b" }
+sp-io = { git = "https://github.com/cennznet/substrate", rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b", default-features = false  }
+crml-support = { path = "../support", default-features = false }
+pallet-scheduler = { git = "https://github.com/cennznet/substrate", default-features = false, rev = "6ec19e279f8b47dabe39640ad93dfbf391d1e48b" }
```

Comment:
another 'dev-dependencies' since its only required for tests

Reply:
This is required by runtime so I think needs to stay where it is